sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogVersionsSuite.scala

```diff
@@ -244,11 +244,16 @@ class HiveExternalCatalogVersionsSuite extends SparkSubmitTestUtils {
       "--conf", s"${WAREHOUSE_PATH.key}=${wareHousePath.getCanonicalPath}",
       "--driver-java-options", s"-Dderby.system.home=${wareHousePath.getCanonicalPath}",
       unusedJar.toString)
+    // TODO After SPARK-46400, we need to run the following logic
     if (PROCESS_TABLES.testingVersions.nonEmpty) runSparkSubmit(args)
   }
 }
 
 object PROCESS_TABLES extends QueryTest with SQLTestUtils {
+  // TODO After SPARK-46400 and applying it to the already released Spark version,
```

Comment:
hmm... I think maybe we can describe it like
```
In SPARK-46302, the env SKIP_SPARK_RELEASE_VERSIONS has been added to allow Maven tests to skip problematic release versions. Related issues will be fixed in SPARK-46400, and testing will be resumed after the fixed Spark 3.x version is released.
```
and I think just add comments here, no need to add them in line 247

Reply:
Okay, let's run this test first.

in addition, before we release `3. x`, should we test `test("backward compatibility")` after the `SPARK-46400` feature is fixed on the branch master?

Reply:
I don't think it has ever tested master...

Reply:
Before the release of the fixed version, Maven's daily tests can skip this case first.